src/@types/third-party.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'storybook-router' {
+  global {
+    interface Window {
+      location: object;
+    }
+  }
+}
```

Comment:
Can we remove storybook from this project?

Reply:
I didn't add it, but without this type file I couldn't compile, so was unsure how to fix!

Reply:
What are you doing to compile/what error do you see? I am able to run `tsc` to compile

Reply:
It happened when I first checked it out:

```
➜  martian git:(upgrade-fix) ✗ npm run compile

> @instantish/martian@1.0.3 compile
> tsc

../node_modules/@types/reach__router/index.d.ts:14:49 - error TS2304: Cannot find name 'Window'.

14 export type WindowLocation<S = LocationState> = Window['location'] & HLocation<S>;
                                                   ~~~~~~


Found 1 error.

npm ERR! code 1
npm ERR! path /Users/cliftoncunningham/work/martian
npm ERR! command failed
npm ERR! command sh -c tsc
```

Reply:
Hmm I cloned your fork, deleted the storybook references, and was able to compile. I notice the error is coming from `node_modules/@types/reach__router/` but when I install the dependencies for this project, that does not get added to my local node modules

Reply:
Very strange.  I even tried deleting `node_modules`, installing a clean node (upgrading to 17), and re-installing.  It's still there on mine.  No idea what to do or if this impacts others?

Reply:
Whats even crazier, is if I run `yarn why @types/reach__router` ... 

```
➜  martian git:(upgrade-fix) ✗ yarn why @types/reach__router
yarn why v1.22.5
[1/4] 🤔  Why do we have the module "@types/reach__router"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
error We couldn't find a match!
✨  Done in 0.36s.
```

Reply:
When you say `deleted the storybook references` ... what do you mean?

Reply:
that is weird! I deleted the file src/@types/third-party.d.ts and the line `"src/@types/*",` from tsconfig.json